src/falcon/falcon_c/rpo.c

```diff
@@ -0,0 +1,757 @@
+/*
+ * RPO implementation.
+ *
+ */
+
+#include <stdint.h>
+#include <string.h>
+
+#define P 0xFFFFFFFF00000001
+
+// From https://github.com/ncw/iprime/blob/master/mod_math_noasm.go
+uint64_t add_mod_p(uint64_t a, uint64_t b)
```

Comment:
Would be great to add section separators in this file (either our usual separators or C-styles ones are fine). For example, this section is for modular arithmetic.

Reply:
Done!